pkgs/applications/kde/kdevelop/kdevelop.nix

```diff
@@ -1,21 +1,18 @@
-
-{ mkDerivation, lib, fetchurl, cmake, gettext, pkg-config, extra-cmake-modules
+{ mkDerivation, lib, cmake, gettext, pkg-config, extra-cmake-modules
 , qtquickcontrols, qtwebkit, qttools, kde-cli-tools, qtbase
 , kconfig, kdeclarative, kdoctools, kiconthemes, ki18n, kitemmodels, kitemviews
 , kjobwidgets, kcmutils, kio, knewstuff, knotifyconfig, kparts, ktexteditor
 , threadweaver, kxmlgui, kwindowsystem, grantlee, kcrash, karchive, kguiaddons
 , plasma-framework, krunner, kdevelop-pg-qt, shared-mime-info, libkomparediff2
-, libksysguard, konsole, llvmPackages, makeWrapper, kpurpose, boost
+, libksysguard, konsole, llvmPackages_13, makeWrapper, kpurpose, boost
+, qtwebengine, cppcheck
 }:
 
+let
+  llvmPackages = llvmPackages_13;
+in
```

Comment:
can we do this where the callPackage for this is? If not easily then this is also fine.

Reply:
I tried it at first (to keep same interface as from the derivation before renaming), but its callPackage is in `kde/default.nix`, and since this is a newScope/function from `libsForQt5`, I couldn't find a `pkgs` reference there, so I had to hardcode the version. Also, couldn't find any other kde desktop/frameworks/gear package allowing overrides, to use as example. If you can point me to some reference, I'll be happy to implement it.